generators/csharp/sdk/src/generator-agent/ReadmeSnippetBuilder.ts

```diff
@@ -0,0 +1,180 @@
+import { FernGeneratorCli } from "@fern-fern/generator-cli-sdk";
+import { FernGeneratorExec } from "@fern-fern/generator-exec-sdk";
+import { EndpointId, FeatureId, FernFilepath, HttpEndpoint } from "@fern-fern/ir-sdk/api";
+import { SdkGeneratorContext } from "../SdkGeneratorContext";
+import { AbstractReadmeSnippetBuilder } from "@fern-api/generator-commons";
+import { EndpointSnippetsGenerator } from "../endpoint/snippets/EndpointSnippetsGenerator";
+
+interface EndpointWithFilepath {
+    endpoint: HttpEndpoint;
+    fernFilepath: FernFilepath;
+}
+
+export class ReadmeSnippetBuilder extends AbstractReadmeSnippetBuilder {
```

Comment:
@amckinney can we make the file structure here 
```
readme/
  ReadmeConfigBuilder.ts
  ReadmeSnippetBuilder.ts
```

Reply:
and move `CsharpSdkGeneratorAgent.ts` to the top?

Reply:
Great call, this reads a lot better. Edited both C# and TypeScript.